src/sentry/incidents/endpoints/serializers.py

```diff
@@ -59,6 +59,41 @@
 CRITICAL_TRIGGER_LABEL = "critical"
 WARNING_TRIGGER_LABEL = "warning"
 
+# TODO: This is temporarily needed.
+# Use a function from discover to break the aggregate down into parts, and then compare the "field"
+# to a list of accepted fields, or a list of fields we need to translate.
+# This can be dropped once snuba can handle this aliasing.
+SUPPORTED_COLUMNS = [None, "tags[sentry:user]", "transaction.duration"]
+TRANSLATABLE_COLUMNS = {
+    "user": "tags[sentry:user]",
+    "dist": "tags[sentry:dist]",
+    "release": "tags[sentry:release]",
+}
+
+
+def forward_translate_snuba_field(aggregate):
```

Comment:
I think it might make sense to split up the "is this a supported aggregate column" from "translate this column". Since we'll need to do this for queries as well then we'll be able to use the "translate this column" logic in both places.